### PR TITLE
docs: update logo

### PR DESCRIPTION
**Pull request by andrewrynhard**
Changes the logo and reformats the description on the front page.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/5104)

### CODE REVIEW

**Review by steverfrancis — APPROVED** [review]

File: website/src/pages/Index.vue [hunk 1]
```diff
@@ -13,10 +13,20 @@
             </p>
 
             <p>
```
Comment (andrewrynhard) [inline website/src/pages/Index.vue:15]: I think I need to remove the paragraph block

**Comment by smira** [comment]
Please don't remove the old logo just yet, we most probably reference it from the conformance results.

**Comment by smira** [comment]
Also need to update here: https://github.com/talos-systems/talos/blob/f49f40a3361381e51d6986547be12ec3b4a3f24a/pkg/cluster/sonobuoy/product.go#L28

**Comment by andrewrynhard** [comment]
> Also need to update here:
> 
> https://github.com/talos-systems/talos/blob/f49f40a3361381e51d6986547be12ec3b4a3f24a/pkg/cluster/sonobuoy/product.go#L28

Looks like I already did.

**Review by smira — APPROVED** [review]

**Pull request merged**